.circleci/config.yml

```diff
@@ -101,6 +101,32 @@ commands:
               circleci-agent step halt
             fi
 
+  webpack_build:
+    # This step represents the build process. It is meant to help developers
+    # detect build related issues in the config file. It does not affect
+    # performance.
+    description: "Build Javascript files"
+    parameters:
+      target_env:
+        type: string
+    steps:
+      - when:
+          condition:
+            equal: [ "prod", << parameters.target_env >> ]
+          steps:
+            - run:
+                name: Builds Javascript files (prod mode)
+                command: |
+                  python -m scripts.run_e2e_tests --deparallelize_terser --prod_env --build-only
```

Comment:
Why are we running `run_e2e_tests` script for building files? should we refactor the build.py such that we can run it from here?

Reply:
The build script currently does not execute webpack build process in dev mode. Also, in `run_e2e_tests` we have some logic to retry webpack builds. But I agree that it can be refactored such that this logic is moved to the build script. Good suggestion!